### DISCUSSION TITLE
paper settings

### DISCUSSION THREAD

**Discussion by i-make-robots** [MarginallyClever/Makelangelo-software#375] [Ideas]
Rotation I threw in, not sure it was ever needed.  there's no way to set negative margin.  One day it would be nice to have a "split picture into n pieces" so they tile to make larger art.

**Comment by PPAC37** [comment]
I have a lot of code reading to do. 
The rotation of the paper (other than 90 ° / landscape) leads me to wonder what are the constraints and limitations of the existing model set up. (As for the PaperSize model and implementation modification for multiple labeled custom size.)

"split picture into n pieces" is a lovely idea !

Polargraph on a roll of paper ( additional axis ) ?

**Comment by i-make-robots** [comment]
I was only thinking of NxM sections for 2A0 and larger drawings.  I'll entertain any idea if you'll write the code :)
